src/vivarium_gates_mncnh/components/intrapartum.py

```diff
@@ -8,17 +8,15 @@
 from vivarium_gates_mncnh.constants.data_values import (
     COLUMNS,
     CPAP_ACCESS_PROBABILITIES,
-    DELIVERY_FACILITY_TYPE_PROBABILITIES,
     DELIVERY_FACILITY_TYPES,
-    PREGNANCY_OUTCOMES,
     SIMULATION_EVENT_NAMES,
 )
 from vivarium_gates_mncnh.utilities import get_location
 
 
 class Intrapartum(Component):
-    """This component functions to make key decisions about events in the intrapartum step of the model.
-    Specifically, it determines the delivery facility type and whether or not CPAP is available to the simulant."""
+    """Component for creating columns necessary for decisions made on time steps that are
```

Comment:
These columns should just be created by the components that are updating them so each column has a single component that is responsible for its contents. I don't see why this components is needed.